goblet/tests/test_vpcconnector.py

```diff
@@ -0,0 +1,146 @@
+from goblet import Goblet
+from goblet.test_utils import get_response, get_responses, dummy_function
+
+# from goblet.infrastructures.vpcconnector import VPCConnector
+# from goblet.resources.http import HTTP
+
+
+class TestVPCConnector:
+    def test_add_vpcconnector(self):
+        app = Goblet(function_name="goblet_example")
+
+        app.vpcconnector(name="vpc-test")
+        vpc = app.infrastructure["vpcconnector"]
+        assert vpc.resources["name"] == "vpc-test"
+
+    def test_deploy_vpcconnector(self, monkeypatch):
+        monkeypatch.setenv("GOOGLE_PROJECT", "goblet")
+        monkeypatch.setenv("GOOGLE_LOCATION", "us-central1")
+        monkeypatch.setenv("GOBLET_TEST_NAME", "vpcconnector-deploy")
+        monkeypatch.setenv("GOBLET_HTTP_TEST", "REPLAY")
+
+        app = Goblet(function_name="goblet-example")
+        app.vpcconnector(name="vpc-test")
+
+        # app.deploy(
```

Comment:
is this commented out because of the vpc client issues?

Reply:
Yes the [Discovery Document list](https://developers.google.com/discovery/v1/reference/apis/list) needs to be updated, still waiting for an update on the feature request. https://github.com/googleapis/google-api-python-client/issues/1913.

Would allow the [current local file](https://github.com/goblet/goblet/blob/main/goblet/tests/data/http/discovery/get-discovery-rest_1.json.bz2) to be updated without manually changing. (it will also break some tests due to updated apis)